ENOENT file or directory does not exist, rename in 1.7.1

Getting an ENOENT for a non existent .tar.gz in the bower cache on Windows 7

1.7.0 works for Pulp (see the comments and build logs in the [aforementioned pull request](https://github.com/bodil/pulp/pull/139)); 1.7.1 doesn't, for reasons I haven't been able to figure out.

A typical error I'm seeing sporadically when bower installs from cache, in case the [build logs](https://ci.appveyor.com/project/bodil/pulp/build/1.0.137) go away:
```
bower purescript-invariant#^0.3.0                    download https://github.com/purescript/purescript-invariant/archive/v0.3.0.tar.gz
bower purescript-st#^0.1.0                            extract archive.tar.gz
bower purescript-control#~0.3.2                       extract archive.tar.gz
bower purescript-node-buffer#^0.2.0                   extract archive.tar.gz
events.js:141
      throw er; // Unhandled 'error' event
      ^
 
Error: ENOENT: no such file or directory, rename 'C:\Users\appveyor\AppData\Local\Temp\1\APPVEYOR-VM-appveyor\bower\4f243f7da48cf97c3bbf39827c643ab7-1416-LAabFw\archive.tar.gz.2110890563' -> 'C:\Users\appveyor\AppData\Local\Temp\1\APPVEYOR-VM-appveyor\bower\4f243f7da48cf97c3bbf39827c643ab7-1416-LAabFw\archive.tar.gz'
    at Error (native)
 
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.2.3
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! pulp@6.1.0 prepublish: `bower install && npm run build`
npm ERR! Exit status 1
```

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "configstore": "^0.3.2",
     "decompress-zip": "^0.1.0",
     "destroy": "^1.0.3",
-    "fs-write-stream-atomic": "^1.0.5",
+    "fs-write-stream-atomic": "1.0.5",
     "fstream": "^1.0.3",
     "fstream-ignore": "^1.0.2",
     "github": "^0.2.3",

